_posts/2015-12-08-week-in-review.md

```diff
@@ -0,0 +1,33 @@
+---
+layout:     post
+title:      Slack Highlights
+date:       2015-12-08
+author:     Max Wofford
+summary:    Here's a few highlighted moments from Hack Club's Slack from the past week
+---
+
+_This post is inspired by the format of
+[Hacker News Highlights](https://themacro.com/articles/2015/12/hacker-news-highlights-2/)._
+
+**Here's a few highlighted moments from Hack Club's Slack from the past week.**
+
+--------------------------------------------------------------------------------
+
+A conversation between a club leader and a high school teacher about the utility
+of significant figures.
+
+> [![](http://i.imgur.com/cOPzC72.png)](https://starthackclub.slack.com/archives/random/p1448939291005835)
+
+--------------------------------------------------------------------------------
+
+Alex and Jonathan hopped on a call to build
+[a tool to track our daily slack activity](https://github.com/jonleung/slack-stats-viewer).
+
+> [![](http://i.imgur.com/ivhjVGE.png)](https://starthackclub.slack.com/archives/random/p1449212126007386)
+
+--------------------------------------------------------------------------------
+
+Our first club leader in Estonia posted conformation of their first club meeting
+on Friday.
+
+> [![](http://i.imgur.com/jRHRA4k.png)](https://starthackclub.slack.com/archives/random/p1449264372007573)
```

Comment:
Actually, instead of using Imgur, what if we put the assets in the `assets` directory of this repository (like we did in the first post)?

https://github.com/hackclub/blog/tree/master/assets